cmd/workspace/lakeview/lakeview.go

```diff
@@ -155,38 +150,36 @@ func newCreateSchedule() *cobra.Command {
 	cmd := &cobra.Command{}
 
 	var createScheduleReq dashboards.CreateScheduleRequest
+	createScheduleReq.Schedule = &dashboards.Schedule{}
 	var createScheduleJson flags.JsonFlag
 
 	// TODO: short flags
 	cmd.Flags().Var(&createScheduleJson, "json", `either inline JSON string or @path/to/file.json with request body`)
 
-	cmd.Flags().StringVar(&createScheduleReq.DisplayName, "display-name", createScheduleReq.DisplayName, `The display name for schedule.`)
-	cmd.Flags().Var(&createScheduleReq.PauseStatus, "pause-status", `The status indicates whether this schedule is paused or not. Supported values: [PAUSED, UNPAUSED]`)
+	cmd.Flags().StringVar(&createScheduleReq.Schedule.CreateTime, "create-time", createScheduleReq.Schedule.CreateTime, `A timestamp indicating when the schedule was created.`)
+	cmd.Flags().StringVar(&createScheduleReq.Schedule.DashboardId, "dashboard-id", createScheduleReq.Schedule.DashboardId, `UUID identifying the dashboard to which the schedule belongs.`)
+	cmd.Flags().StringVar(&createScheduleReq.Schedule.DisplayName, "display-name", createScheduleReq.Schedule.DisplayName, `The display name for schedule.`)
+	cmd.Flags().StringVar(&createScheduleReq.Schedule.Etag, "etag", createScheduleReq.Schedule.Etag, `The etag for the schedule.`)
+	cmd.Flags().Var(&createScheduleReq.Schedule.PauseStatus, "pause-status", `The status indicates whether this schedule is paused or not. Supported values: [PAUSED, UNPAUSED]`)
+	cmd.Flags().StringVar(&createScheduleReq.Schedule.ScheduleId, "schedule-id", createScheduleReq.Schedule.ScheduleId, `UUID identifying the schedule.`)
+	cmd.Flags().StringVar(&createScheduleReq.Schedule.UpdateTime, "update-time", createScheduleReq.Schedule.UpdateTime, `A timestamp indicating when the schedule was last updated.`)
 
-	cmd.Use = "create-schedule DASHBOARD_ID"
+	cmd.Use = "create-schedule"
```

Comment:
Is the ID a path parameter?

Reply:
Changed PR to respect path parameters, fixed now